cocos/dragon-bones/ArmatureSystem.ts

```diff
@@ -0,0 +1,71 @@
+import { director, Director } from '../core/director';
+import { System } from '../core/components';
+import { ArmatureDisplay } from './ArmatureDisplay';
+import { legacyCC } from '../core/global-exports';
+
+export class ArmatureSystem extends System {
+    /**
+     * @en
+     * The ID flag of the system.
+     * @zh
+     * 此系统的 ID 标记。
+     */
+    static readonly ID = 'ARMATURE';
+
+    /**
+     * @en
+     * Gets the instance of the tween system.
+     * @zh
+     * 获取Spine骨骼系统的实例。
+     */
+
+    public static get instance (): ArmatureSystem {
+        return ArmatureSystem._instance;
+    }
+
+    public static set instance (sys:ArmatureSystem) {
```

Comment:
Not necessary, singleton instance don't need to be overridden. You need to check whether `_instance` exist, if not, create it automatically